### DISCUSSION TITLE
Bitdefender GravityZone API

### DISCUSSION THREAD

**Discussion by rlabaki** [grafana/grafana-infinity-datasource#742] [Q&A]
I am trying to integrate GravityZone Bitdefnder API in Grafana through Infinity

For the API 
The Authorization header consists of the following elements:
The authorization method and a space as the prefix; in our case, this will always be equal to Basic.
A Base64 encoded string, generated from the combined username:password sequence.

So i need to configure Infinity with only a user name and  i need to leave the password field empty 
is there any way to do that ?

Thx

**Comment by yesoreyeram** [comment]
If you pass the username and password, and set to basic authentication infinity will automatically handle the base64 thing for you.

Other option is you can set the auth method to no auth and then add the Authorization header directly.
